deploifai/server/create.py

```diff
@@ -0,0 +1,386 @@
+import click
+from PyInquirer import prompt
+
+from deploifai.context import pass_deploifai_context_obj, DeploifaiContextObj, is_authenticated, project_found
+from deploifai.utilities.version_comparison import compare, multi_compare
+from deploifai.utilities.python_supported import supported
+
+framework_options = ["tensorflow", "pytorch"]
+server_options = ["SMALL", "MEDIUM", "LARGE"]
+
+
+@click.command()
+@click.argument("name")
+@click.option("--no-dataset", is_flag=True, default=False, show_default=True,
+              help="No datasets to be connected to server")
+@click.option("--dataset", "-d", type=str, help="Name of dataset to be used")
+@click.option("--cloud-profile-name", "-c", type=str, help="Name of cloud profile to be used")
+@click.option("--gpu/--no-gpu", default=True, show_default=True, help="Create training server with or without GPU")
+@click.option("--no-framework", is_flag=True, default=False, show_default=True,
+              help="No frameworks to be installed in the server")
```

Comment:
`Do not install ML frameworks in the server`

Reply:
changed